src/LanguageServer/Impl/Sources/DefinitionSource.cs

```diff
@@ -286,7 +286,19 @@ internal abstract class DefinitionSourceBase {
                         return FromMember(v1);
                     }
                     break;
-
+                case IPythonSuperType cls:
+                    // Data members may be PythonInstances which do not track their declaration location.
+                    // In this case we'll try looking up the respective variable instead according to Mro.
+                    foreach (var b in cls.Mro.OfType<IPythonClassType>()) {
+                        using (eval.OpenScope(b)) {
+                            eval.LookupNameInScopes(mex.Name, out _, out var v2, LookupOptions.Local);
+                            if (v2 != null) {
+                                definingMember = v2;
+                                return FromMember(v2);
+                            }
+                        }
+                    }
+                    break;
```

Comment:
This is the code needed for gotodefinition to work for super().v   its just a copy of the code below

Reply:
I could make a new interface that combines IPythonClassType and IPythonSuperType

Reply:
If the only thing in common for these is the MRO, then maybe an MRO is something that every type should have? Spitballing.

Reply:
If it were me I'd stick with a little bit of copying and a TODO rather than trying to abstract this immediately.